src/ws/cockpitauth.c

```diff
@@ -511,7 +511,8 @@ static CockpitCreds *
 create_creds_for_spawn_authenticated (CockpitAuth *self,
                                       const char *user,
                                       SpawnLoginData *sl,
-                                      JsonObject *results)
+                                      JsonObject *results,
+                                      const char *raw_data)
```

Comment:
Why are `user` and `raw_data` in this function of type `const char *` and not `const gchar *`? The values passed to them are `gchar`.